apps/right-to-rent-check/acceptance/features/tenancy-start.js

```diff
@@ -6,31 +6,36 @@ const steps = require('../../');
 Feature('Tenancy Start Page');
 
 Before((
-  I,
-  tenancyStartPage
+  I
 ) => {
-  I.visitPage(tenancyStartPage, steps);
+  I.amOnPage('/');
 });
 
 Scenario('The correct fields are on the page', (
   I,
   tenancyStartPage
 ) => {
+  I.completeToStep('/tenancy-start', {
```

Comment:
Also, if you use `I.visitPage` in the `beforeEach` you should not need to do `completeStep` at all.